### PR TITLE
Leaderboard API

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
- In this PR I created a leaderboard using the Microverse Leaderboard API. The project used Webpack and ES6 modules to enhance functionality. It allows the user to add their name and scores which display on the list when refreshed.

**Highlights** 

- Setup project with Webpack
- Setup Linters and correct errors
- Use ES6 modules to organize JavaScript code
- Style the page using flexbox 

**Cheers and Happy Reviewing!**

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi @sharon-odhiambo , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You have submitted a project, that does not have a descriptive README file.** 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
 // import _ from 'lodash';
```
Comment (rustam-tolipov) [inline src/index.js]: - [ ] [OPTIONAL] Please remove unused commented code 👌🏼

File: README.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Big thanks to the microverse community with a special mention of my coding partners, morning session teams, standup teams and reviewers.
+## 📝 License
+This project is [MIT](./LICENSE.txt) licensed.
```
Comment (rustam-tolipov) [inline README.md:60]: - [ ] [OPTIONAL] Please kindly create MIT license 📃

**Review by rustam-tolipov — APPROVED** [review]
## Hello @sharon-odhiambo 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**